aea/helpers/profiling.py

```diff
@@ -194,6 +218,45 @@ def get_objects_instances(self) -> Dict:
             lock.release()
         return result
 
-    def get_objecst_created(self) -> Dict:
+    def get_objects_created(self) -> Dict:
         """Return dict with counted object instances created."""
         return self._counter
+
+
+def get_dialogues_by_type() -> Dict:
+    """Return dict with the number of dialogues by type."""
+
+    seen_dialogue_ids: List[int] = []
```

Comment:
These variables are bound to the scope of this method, below we iterate a list `gc.get_objects()`, so what is this useful for?

Reply:
I think they are not useful anymore. They were there because at some point the method also iterated over every object's attributes (trying to measure size) and we had the risk of counting the same objects several times, so these lists helped us to keep track of already seen objects. Since now we have just one iteration, I can delete them.